### ISSUE TITLE
UserGuide: wrong anchor for NVDA 2024.3beta5

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#16948]
### Actual behavior:
In NVDA 2024.3beta5, opening the english userGuide, system requirements has wrong anchor. anchors refer both to minimumSystemRequirements, instead the first should be Recommended.
Current code:

`<code>`
<li><a href="#SystemRequirements" title="System Requirements">System Requirements</a><ul>
<li><a href="#MinimumSystemRequirements" title="Recommended System Requirements">Recommended System Requirements</a></li>
<li><a href="#MinimumSystemRequirements" title="Minimum System Requirements">Minimum System Requirements</a></li>
</ul>
`</code>`

### Expected behavior:
Code should be as following:
`<code>`
<li><a href="#SystemRequirements" title="System Requirements">System Requirements</a><ul>
<li><a href="#RecommendedSystemRequirements" title="Recommended System Requirements">Recommended System Requirements</a></li>
<li><a href="#MinimumSystemRequirements" title="Minimum System Requirements">Minimum System Requirements</a></li>
</ul>
`</code>`

#### NVDA version:
NVDA 2024.3beta5

**Comment by dreinn** [comment]
ok sorry, code doesn't appear normally, but I hope at least concept is clear

**Comment by seanbudd** [comment]
Hi, thanks for reporting



Closing as duplicate of #16941
